int32 data
---
bool success   # indicate successful run of service
string message # informational